### DISCUSSION TITLE
Auto-update corejs in env preset?

### DISCUSSION THREAD

**Discussion by inwardmovement** [babel/babel#11719] [Q&A] [answered]
Do we have to manually update [corejs version](https://babeljs.io/docs/en/babel-preset-env#corejs) (in the `env` preset) each time the `corejs` package gets update in `package.json` (3.x.x)?

Can't we use something like the following?
```
"@babel/preset-env",
{
  "corejs:": "^3"
}
```

**Comment by nicolo-ribaudo** [comment, marked as answer]
You can use `^3.0.0`, however it means that Babel's output should be compatible with every core-js version starting from 3.0.0.

What you probably want is something like this:
```js
// babel.config.js
module.exports = {
  presets: [
    ["@babel/preset-env", {
        corejs: require("./package.json").dependencies["core-js"]
    }]
};
```

**Reply by inwardmovement** [reply]
Thanks!

**Reply by SharakPL** [reply]
And here I've thought this is all I need

```js
// babel.config.js
module.exports = {
  presets: [
    ["@babel/preset-env", {
        corejs: 3
    }]
};
```
Thanks @nicolo-ribaudo
